lib/web_ui/lib/src/engine/text_editing.dart

```diff
@@ -741,6 +807,19 @@ class HybridTextEditing {
       ..transform = transformCss;
   }
 
+  // TODO(flutter_web): When the browser closes and re-opens the virtual
+  // shifts the page in IOS. Call this method from visibility change listener
+  // attached to body.
```

Comment:
Why is the above comment a `TODO`?

Reply:
See also: https://github.com/flutter/flutter/wiki/Style-guide-for-Flutter-repo#avoid-checking-in-comments-that-ask-questions

Reply:
Cause it will take anoter 50+lines. This PR is already to big. It is a TODO for a next PR :)

Reply:
It is not a question by the way, when meant as a conjunction. I changed when with after.